### ISSUE TITLE
Request: Details View next to Timline

### ISSUE THREAD

**Issue by Montrazul** [richardtop/CalendarKit#57]
It would be nice to have a customizable details view next to the timeline view. For now i just cant get it working. I tried the following in the `DayView`'s `layoutSubviews`:

```
self.dayViewHeader.anchorAndFillEdge(.top, xPad: 0, yPad: 0, otherSize: self.dayViewHeaderHeight);
// self.timelinePager.alignAndFill(align: .underMatchingLeft, relativeTo: self.dayViewHeader, padding: 0);
        
self.timelinePager.alignAndFillHeight(align: .underMatchingLeft, relativeTo: self.dayViewHeader, padding: 0, width: ((self.frame.width / 3) * 2));
self.dayViewDetails.alignAndFillHeight(align: .underMatchingRight, relativeTo: self.dayViewHeader, padding: 0, width: (self.frame.width / 3));
```
But this is really messing up everything :D

**Comment by richardtop** [comment]
Hi @Montrazul,
could you please, attach a mockup of the result you'd like to achieve?
CalendarKit is built using [Neon](https://github.com/mamaral/Neon) to define layout. You may need to read Neon's docs to understand some of its concepts.

As soon as I could see the desired result, I'll be more than welcome to help.

**Comment by Montrazul** [comment]
Yes ofcourse. Thats the goal:

http://i.imgur.com/O8jmf6a.jpg

I already was able to build the layout with neon like it is on the picture but the problem is when I do that the paging scroll view is totally messed up. The TimelineView with is too much, the scroll offset isnt working etc. I guess it is because all the views are initialized with the full width of the view and I have no idea how to fix it. I already played around a little with numbers but I did not get it working.

**Comment by richardtop** [comment]
@Montrazul this is a custom layout, I'd recommend creating it from scratch from CalendarKit components and not using DayView.
So, what I propose, is to treat your layout as a custom variant of DayView.
To build it, you could take `DayHeaderView` and `timelinePager` - a set of `TimelineContainer`'s embedded in `PagingScrollView`.

Here is the illustration, which components you could reuse:
<img width="778" alt="screen shot 2017-05-21 at 20 16 20" src="https://cloud.githubusercontent.com/assets/8013017/26286026/1bc248c6-3e64-11e7-9f08-93bdfa2df41a.png">
Unfortunately, there is no way to extract violet module, so you have to construct it on your own (see, how it's made in DayView).

So, to sum up, to build the view you posted, try creating it from scratch using CK components instead of modifying current DayView.
Feel free to ask any questions and post your results here.

**Comment by richardtop** [comment]
Now it is possible to extract module colored in violet, a change made in #64.
